cmd/test.go

```diff
@@ -23,8 +23,14 @@ var (
 	}
 )
 
+var (
+	testDir string
+)
+
 func init() {
 	RootCmd.AddCommand(TestCmd)
+
+	TestCmd.Flags().StringVar(&testDir, "test-dir", "integration", "Name of the directory containing executable tests.")
```

Comment:
Adding a test dir flag which defaults to the currently used value. That way we can structure different tests in different directories and run them in parallel OOB by plain circle support.